views/partials/footer.php

```diff
@@ -0,0 +1,3 @@
+  </div>
+</body>
+</html>
```

Comment:
This feels awkward. We should be able to remove the footer component without breaking the structure of our main html template.